AGA8CODE/C/Detail.cpp

```diff
@@ -1,5 +1,3 @@
- 
```

Comment:
Can you revert this whitespace-only change?

Reply:
Done.